src/prefect/cli/deploy.py

```diff
@@ -233,7 +234,16 @@ async def deploy(
     }
     try:
         deploy_configs, actions = _load_deploy_configs_and_actions(ci=ci)
-        deploy_configs = _pick_deploy_configs(deploy_configs, names, deploy_all, ci)
+
```

Comment:
minor nit: should we add a note in the CLI help text about the newly supported input formats?

Reply:
> This looks scary powerful! So of course I'm excited to try it!

Please do! QA on `main` street tomorrow

Reply:
>  should we add a note in the CLI help text about the newly supported input formats?

yes, thanks, I can add a note!